src/tv2-common/migrations/index.ts

```diff
@@ -173,6 +174,74 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string, fromValue: string, targetTable: string) {
```

Comment:
The name of this migration specifies that it has a very precise job to do i.e. moving `SelectedGfxSetupName` to `GfxDefaults` - do you then really need to have a `fromValue` and a `targetTable` as parameters?
Do you plan on reusing this migration?

By not having the parameters you could make some more precise names for your locale variables. `singleValue` could then be `selectedGfxEntry` or something similar.

Reply:
Removed!